.travis.yml

```diff
@@ -1,8 +1,6 @@
 sudo: false
 language: node_js
 env: TEST_SUITE=unit
-node_js:
-  - "5.6.0"
```

Comment:
i prefer putting 6.3.0 here instead of `.nvmrc` just because it will save a lot of pain in the future (next time Travis breaks, people will immediately open `.travis.yml`, and it may not be obvious to someone else that they need to look at `.nvmrc`).